modules/ml/src/main/java/org/apache/ignite/ml/dataset/PartitionDataBuilder.java

```diff
@@ -48,6 +54,10 @@
      */
     public D build(Iterator<UpstreamEntry<K, V>> upstreamData, long upstreamDataSize, C ctx);
 
+    default public D build(Stream<UpstreamEntry<K, V>> upstreamData, long upstreamDataSize, C ctx) {
```

Comment:
Missed javadoc as far as I see. Plus I think that the write approach is to write "public default", not "default public".